Adicionando instrução para renomear arquivo do model

JVictorV opened the pull request.

wallmartins commented:
Salve, 7h3r4v3n, tudo tranquilo?

Então, as regras do he4rtoberfest pedia para que fosse criada uma branch seguindo o padrão challenge/challenge-id/lang/nickname e, no caso do seu PR, você realizou o commit direto na master, dessa forma, precisarei pedir para que você refaça o seu PR seguindo esse formato para que a gente possa aceitá-lo e ele venha a ser revisado pela galera, tranquilo?

Tem o passo a passo nas instruções iniciais de como realizar a criação da branch e o commit para ela, mas qualquer dúvida, pode perguntar lá na He4rt que a galera ajuda.

JVictorV committed:
Merge remote-tracking branch 'upstream/master'

JVictorV committed:
Merge remote-tracking branch 'upstream/master'

JVictorV committed:
Adicionando instrução para renomear arquivo do model

Como solicitado em alguns PRs o model.md deve ser renomeado para readme.md ao ser adicionado na pasta do desafio. Como não encontrei essa instrução no readme, acabei adicionando ao sétimo passo

Logikoz reviewed (approved):
Nice man.
